builder/vmware/common/driver.go

```diff
@@ -600,3 +603,28 @@ func (d *VmwareDriver) HostIP(state multistep.StateBag) (string, error) {
 	}
 	return "", fmt.Errorf("Unable to find host IP from devices %v, last error: %s", devices, lastError)
 }
+
+func GetOVFTool() string {
+	ovftool := "ovftool"
+	if runtime.GOOS == "windows" {
+		ovftool = "ovftool.exe"
+	}
+
```

Comment:
Maybe not for this scope but sometimes I feel like we should create an ovftool package that does all of this.

Reply:
🤔 maybe. Feels out of scope here though, yeah.